src/ray/core_worker/reference_count.cc

```diff
@@ -599,6 +599,15 @@ void ReferenceCounter::ReleasePlasmaObject(ReferenceTable::iterator it) {
     it->second.on_delete = nullptr;
   }
   it->second.pinned_at_raylet_id.reset();
+  if (it->second.spilled) {
+    // The spilled copy of the object should get deleted during the on_delete
+    // callback, so reset the spill location metadata here.
+    // TODO(swang): Spilled copies in cloud storage are not GCed, so we should
```

Comment:
Maybe we should just add logic now? Isn't it just spilled & node_id == Nil, do nothing?